### PR TITLE
Add recommended guards helper

### LINKED ISSUE #5: Add recommendedGuards() helper + README snippet
## Context

Acceptance criterion in agent-labs-dev/nebula-desktop#153 calls for a \`recommendedGuards()\` snippet in the README. The helper itself doesn't exist yet — this issue covers both designing it and documenting the call site.

## Scope

- Decide what \`recommendedGuards()\` returns (env opt-in check? \`app.isPackaged\` gate? loopback-only assertion?) — sketch in PRD agent-labs-dev/nebula-desktop#149 if there's prior art
- Implement it in \`src/index.ts\` (or a new \`src/guards.ts\`) and re-export from the public entry
- Add the snippet to README

## Acceptance

- [ ] \`recommendedGuards()\` is callable from \`@nebula-agents/electron-mcp\` and returns either booleans or throws on misconfiguration
- [ ] README has a copy-pasteable snippet showing how a consumer wires it before \`createElectronMcpServer().start()\`

### PR DESCRIPTION
**Pull request by jnsdls**
## Summary

- add public `recommendedGuards()` helper for unpackaged + env-var MCP opt-in checks
- export the helper and its options type from the root and types entrypoints
- update README quickstart/security snippets to use the package helper

## Verification

- [x] `pnpm run lint`
- [x] `pnpm run typecheck`
- [x] `pnpm run test`
- [x] `pnpm run build`
- [ ] `pnpm test:electron` if Electron/CDP behavior changed

Closes #5

## Summary by CodeRabbit

* **New Features**
  * Added a public helper to opt into MCP when the app is unpackaged and a specified environment variable equals "1".
* **Documentation**
  * Quickstart and guidance updated with example usage and clearer behavior and edge-case notes.
* **Tests**
  * Unit tests added covering packaged vs unpackaged states, env-var opt-in, and error conditions.
* **Chores**
  * Changeset added documenting a minor version bump.

### CODE REVIEW

File: src/guards.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+const DEFAULT_ENV_VAR = "ELECTRON_MCP";
+
+export function recommendedGuards({
+  app,
+  isPackaged,
+  env = process.env,
+  envVar = DEFAULT_ENV_VAR,
+}: RecommendedGuardsOptions): boolean {
```
[the remark this answers is not in the document]
Comment (jnsdls) [reply src/guards.ts]: Fixed in 884f455 by defaulting the options object and adding a zero-argument regression test.
